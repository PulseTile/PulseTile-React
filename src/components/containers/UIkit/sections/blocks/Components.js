import React from 'react';
import Buttons from '../fragments/components/Buttons';
import ControlGroup from '../fragments/components/ControlGroup';
import Form from '../fragments/components/Form';
import FormSection from '../fragments/components/FormSection';
import Inputs from '../fragments/components/Inputs';
import Selects from '../fragments/components/Selects';
import Radio from '../fragments/components/Radio';
import Checkboxes from '../fragments/components/Checkboxes';
import Switch from '../fragments/components/Switch';
import InputFile from '../fragments/components/InputFile';
import Datepicker from '../fragments/components/Datepicker';
import SliderRange from '../fragments/components/SliderRange';
import Scrollbar from '../fragments/components/Scrollbar';
import Labels from '../fragments/components/Labels';
import ThemeColors from '../fragments/components/ThemeColors';
import Selectable from '../fragments/components/Selectable';
import Highlighters from '../fragments/components/Highlighters';
import SpinnerBlock from '../fragments/components/Spinner';
import Pagination from '../fragments/components/Pagination';
import Modal from '../fragments/components/Modal';
import Dropdowns from '../fragments/components/Dropdowns';
import Panels from '../fragments/components/Panels/Panels';
import Tables from '../fragments/components/Tables';

/**
 * This component returns content of Component block of UIkit page
 */
const Component = () => {
  return (
    <div id="components" className="ui-section">
      <h2 className="ui-main-title">Components</h2>
      <Buttons />
      <ControlGroup />
      <Form />
      <FormSection />
      <Inputs />
      <Selects />
      <Checkboxes />
      <Radio />
      <Switch />
      <InputFile />
      <Datepicker />
      <SliderRange />
      <Scrollbar />
      <Labels />
      <ThemeColors />
      <Selectable />
      <Highlighters />
      <SpinnerBlock />
      <Pagination />
      <Modal />
      <Dropdowns />
      <Panels />
      <Tables />
    </div>
  );
};

export default Component;
