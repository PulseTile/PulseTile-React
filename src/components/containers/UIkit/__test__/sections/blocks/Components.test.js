import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Components from '../../../sections/blocks/Components';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Components />', () => {
  it('should renders Components with props correctly', () => {
    const component = shallow(<Components />);
    expect(component).toMatchSnapshot();
  });
});
