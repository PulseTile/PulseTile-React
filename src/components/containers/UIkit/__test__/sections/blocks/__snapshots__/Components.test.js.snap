// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Components /> should renders Components with props correctly 1`] = `
<div
  className="ui-section"
  id="components"
>
  <h2
    className="ui-main-title"
  >
    Components
  </h2>
  <Buttons />
  <ControlGroup />
  <Form />
  <FormSection />
  <Inputs />
  <Selects />
  <Checkboxes />
  <Radio />
  <Switch />
  <InputFile />
  <Datepicker />
  <SliderRange />
  <Scrollbar />
  <Labels />
  <ThemeColors />
  <Selectable />
  <Highlighters />
  <SpinnerBlock />
  <Pagination />
  <Modal />
  <Dropdowns />
  <Panels />
  <Tables />
</div>
`;
